apps/server/src/shared/infra/identity-management/keycloak/controller/console/keycloak-management.console.ts

```diff
@@ -115,13 +119,19 @@ export class KeycloakConsole {
 		delay: number
 	): Promise<T> {
 		let repetitions = 0;
+		let error = new Error();
```

Comment:
An error with an empty message is bad. 
This error could be thrown if `count` is <1. Please put that in the error message.